### DISCUSSION TITLE
How to add a key to value

### DISCUSSION THREAD

**Discussion by japgroevemaker** [alpinejs/alpine#3890] [1. Help]
Hello,

When i'm checking the checkbox i'm pushing the value of the checkbox to an array. But how do i add an key to my value?
` <input type="checkbox" id="whitepaper" x-model="selectedFilters[]" value="whitepaper">`

**Comment by ekwoka** [comment]
What do you mean a key? It's a list. Not key value pairs.

The key is `selectedFilters`

**Reply by japgroevemaker** [reply]
When i'm checking my checkbox i want the structure of the array to be something like this:
`
selectedFilters[{ 'subject':  'whitepaper'}]
`

**Reply by ekwoka** [reply]
Seems like you should make `selectedFilters` and object, and model the subject checkboxes to a property on that object that has an array.
